### PR TITLE
feat(rtl-dispaly-support): added rtl table grid and progressbar display support

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: src/exporter/components/export-table/cell-renderer/progress.cell-renderer.tsx [hunk 1]
```diff
@@ -3,13 +3,16 @@ import { ICellRendererParams } from 'ag-grid-community';
...
+import { useState } from 'react';
+import EXPORTER_CONFIG from '../../../../common/config';
 import { IExportTaskStatus } from '../../../models/exportTaskStatus';
 
 const PEDING_PROGRESS = 0;
 const ONE_HUNDRED = 100;
 export const ProgressRenderer: React.FC<ICellRendererParams> = (
   props
 ) => {
+  const [isReversed] = useState<boolean>(EXPORTER_CONFIG.I18N.DEFAULT_LANGUAGE === 'he' ? true : false);
```
Comment (asafMasa) [inline src/exporter/components/export-table/cell-renderer/progress.cell-renderer.tsx:15]: isReversed or isRtl -> use always isRTL (don't use different names for the same meaning)

File: src/exporter/components/export-table/export-table-dialog.tsx [hunk 2]
```diff
@@ -141,6 +142,14 @@ export const ExportSatusTableDialog: React.FC<ExportSatusTableDialogProps> = obs
           },
           suppressMovable: true,
         },
+        {
+          headerName: intl.formatMessage({
+            id: 'export-table.table-column-header.sourceLayer.text',
+          }),
+          width: 170,
+          field: 'sourceLayer',
```
Comment (asafMasa) [inline src/exporter/components/export-table/export-table-dialog.tsx:150]: this code don't belong in this PR - remove it

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/exporter/components/export-table/export-table-dialog.tsx (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply src/exporter/components/export-table/export-table-dialog.tsx]: fixed and removed in dbef340

File: src/exporter/components/export-table/cell-renderer/progress.cell-renderer.tsx (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply src/exporter/components/export-table/cell-renderer/progress.cell-renderer.tsx]: fixed in dbef340

**Review by asafMasa — APPROVED** [review]
